lib/tool-bar-button-view.coffee

```diff
@@ -34,3 +37,14 @@ module.exports = class ToolBarButtonView extends View
 
   destroy: ->
     @subscriptions.dispose()
+
+  getPreviouslyFocusedElement: () ->
+    if @previouslyFocusedElement[0] and @previouslyFocusedElement[0] isnt document.body
+      @eventElement = @previouslyFocusedElement[0]
+    else
+      @eventElement = atom.views.getView(atom.workspace)
+
+  storeFocusedElement: ->
+    activeElement = $(document.activeElement)
+    if !activeElement.hasClass 'tool-bar-btn'
```

Comment:
> If you change this line to if not activeElement.classList.contains 'tool-bar-btn' you can drop jQuery as this isn't really necessary and only causes overhead.

Thanks! I did it :smile: 

Of course I want to switch to vanilla JS as much as possible.